en/contributing/build.md

```diff
@@ -315,6 +315,42 @@ sudo make default install
 ```
 See [SDK Extensions](../guide/sdk_extensions.md) for more information.
 
+## Building the Backend {#build_backend}
+
+The *Dronecode SDK* programming-language-specific libraries (e.g. [Swift](http://dronecode-sdk-swift.s3.eu-central-1.amazonaws.com/docs/master/index.html), [Python](https://github.com/Dronecode/DronecodeSDK-Python#dronecodesdk-python)) share a common backend, which may optionally be built as part of the C++ library.
+
+The build additionally depends on libraries for the *Go* programming language, and the *make* option `BUILD_BACKEND=1`.
+Otherwise the build is exactly the same as usual.
+
+### Ubuntu {#build_backend_ubuntu}
+
+To build the backend on Ubuntu:
+1. [Setup/Build the C++ Library on Linux](#build_cpp_linux)
+1. Install additional dependencies  
+   ```
+   sudo apt-get install golang
+   ```
+1. Navigate into the SDK directory and clean any existing build artifacts
+   ```
+   cd DronecodeSDK
+   make distclean
+   ```
+1. Build the project:
+   ```
+   make BUILD_BACKEND=1
+   ```
+
+
+### macOS {#build_backend_macos}
+
+TBD
+
+### Windows {#build_backend_windows}
+
+TBD
```

Comment:
Hmm :thinking:

Reply:
I suggest we just merge with TBD for now.

Reply:
ok!